### DISCUSSION TITLE
Backup Job returns "failed" despite of all the Volumes being sucessful

### DISCUSSION THREAD

**Discussion by othapall** [longhorn/longhorn#9401] [Q&A]
longhorn version 1.5.3

backup job returns "failed" despite all volumes showing as successful in Longhorn. No error logs found backup job pod logs.

**Comment by PhanLe1010** [comment]
What does it mean by `backup job returns "failed"`? Do you mean the recurring job status is failed? If socCan you also upload the logs from last failed pods?

Can you double check the backup CR to see if there is no backup failed indeed, `kubectl get backups -n longhorn-system -oyaml`

**Reply by othapall** [reply]
I'm sure that no backup failure on any of the volume
